### ISSUE TITLE
Questions & answers 2019

### ISSUE THREAD

**Issue by JeanLucAnton** [Centre-IRM-INT/GT-MVPA-nilearn#1]
Q : Can MVPA be used simply as a way of improving power/sensitivity over univariate methods i.e. given an appropriate experimental design, can MVPA be used to answer the same kinds of questions we ask using univariate methods?  If the answer is yes, then I guess this means that we all potentially have existing data-sets that could be reanalysed using multivariate methods?  If the experiment was originally designed for a univariate analysis, how can we best/fairly split the data into a training set and a test set ?
A : The response seems to be yes, as the use of MVPA tends to increase sensitivity over univariate analyses (Li et al., 2009, Neuron). In the field of motor cognition, for example, Buchwald et al. (2018) reanalyzed with MVPA a data-set already analyzed and published in 2017 with univariate analyses. However, I ignore how easy it is to make MVPA on data recorded with a fMRI design originally created for univariate analyses.

Q : If perception of features A and B are first identified by e.g. localisers, can MVPA then be used to decode whether participants were preferentially processing feature A or feature B during presentation of stimuli comprising both of these features?  Could we then correlate this to behavioural performance?

Q : Can MVPA be used as a way of showing regions that are common to two distinct tasks?  e.g. train the classifier on task A, test it on task B, and see which regions are identified?  Is this more sensitive than a conjunction of two univariate analyses?

Q : How big a role does inter-individual variability play in a classifier’s ability to categorise activations?  Is the classification done on an individual basis, or on a group basis?
A : on an individual basis.  

Q : Could we potentially train a classifier on data from one group of participants and then test it on data from a separate group of participants? 
A : Sylvain says yes, it's possible.  Also, can combine data from several subjects for the classifier - even though it's noisier because of inter-subject variability, the fact that you've increased the number of data-points makes it worth it.
